packages/server/lambdas/alfresco/list-components.ts

```diff
@@ -2,13 +2,14 @@ import { ALBEvent, ALBResult } from 'aws-lambda';
 import { findEndpoint } from '../../utils/alfresco';
 import { getRataExtraLambdaError, RataExtraLambdaError } from '../../utils/errors';
 import { log } from '../../utils/logger';
-import { getUser, validateReadUser, validateWriteUser } from '../../utils/userService';
+import { getUser, validateReadUser } from '../../utils/userService';
 import { DatabaseClient } from '../database/client';
+import { getComponents } from '../database/components/get-components';
 
 const database = await DatabaseClient.build();
 
 /**
- * Get page components. Example request: GET /api/alfresco/folder/linjakaaviot
+ * Get page components. Example request: GET /api/alfresco/folders/linjakaaviot
```

Comment:
Shouldn't this be the other way around now? It was with s but that was removed.

Reply:
Agree, `/folders` is better

Reply:
What I meant was that this was changed to be without s elsewhere, but was changed to contain s here, which was inconsistent.